bundles.rst

```diff
@@ -86,26 +105,24 @@ The directory structure of a bundle is meant to help to keep code consistent
 between all Symfony bundles. It follows a set of conventions, but is flexible
 to be adjusted if needed:
 
-``Controller/``
+``src/Controller/``
```

Comment:
maybe we should change this path to `src/` only and explain what it contains, not only controllers

Reply:
Done.